relays/bin-substrate/src/cli/mod.rs

```diff
@@ -500,11 +531,32 @@ macro_rules! declare_chain_options {
 				/// Convert connection params into Substrate client.
 				pub async fn to_client<Chain: CliChain>(
 					&self,
+					bundle_spec_version: u32,
```

Comment:
Could we replace these two parameters with `Option<RuntimeVersion>` here? `Option` because for some chains (where we won't submit any transaction) the version may be unknown (so it shall be treated as `ChainRuntimeVersion::Auto`). And using `RuntimeVersion` would allow us replace four consts with two, where `to_client()` is called. We'll probably move `VERSION` constants to `Chain` trait later => remove these consts at all.

Reply:
Last q - is there any specific reason you have decided to use two `Option<u32>`arguments instead of single `Option<RuntimeVersion>` as I've suggested here? If not, we'll change this in near future

Reply:
Sorry,  I missing here. Let me think about it. Your suggestions are better. I can change it. Please wait.

Reply:
https://github.com/paritytech/parity-bridges-common/pull/1248/commits/e607a372fb3c71e48e58306d47aa4e73216e0164